poem/Poem/frontend/react/App.js

```diff
@@ -469,511 +469,493 @@ const SuperAdminRouteSwitch = ({props}) => (
 )
 
 
-class App extends Component {
-  constructor(props) {
-    super(props);
-
-    this.cookies = new Cookies();
-    this.backend = new Backend();
-
-    this.state = {
-      isSessionActive: undefined,
-      areYouSureModal: false,
-      userDetails: undefined,
-      webApiAggregation: undefined,
-      webApiMetric: undefined,
-      webApiThresholds: undefined,
-      webApiOperations: undefined,
-      webApiReports: undefined,
-      publicView: undefined,
-      tenantName: undefined,
-      token: undefined,
-      version: undefined,
-      isTenantSchema: null
-    };
-
-    this.onLogin = this.onLogin.bind(this);
-    this.onLogout = this.onLogout.bind(this);
-    this.toggleAreYouSure = this.toggleAreYouSure.bind(this);
-  }
-
-  async onLogin(json) {
+const App = (props) => {
+  const cookies = new Cookies();
```

Comment:
* `props` are never used within component so it can be safely removed
* `cookies` also therefore importing of `Cookies` package can be removed as well